aggregates/governance-aggr/src/main/java/com/bloxbean/cardano/yaci/store/governanceaggr/service/BootstrapPhaseDetector.java

```diff
@@ -0,0 +1,55 @@
+package com.bloxbean.cardano.yaci.store.governanceaggr.service;
+
+import com.bloxbean.cardano.client.common.model.Networks;
+import com.bloxbean.cardano.yaci.core.model.Era;
+import com.bloxbean.cardano.yaci.store.common.config.StoreProperties;
+import com.bloxbean.cardano.yaci.store.common.domain.ProtocolParams;
+import com.bloxbean.cardano.yaci.store.epoch.domain.EpochParam;
+import com.bloxbean.cardano.yaci.store.epoch.processor.EraGenesisProtocolParamsUtil;
+import com.bloxbean.cardano.yaci.store.epoch.storage.EpochParamStorage;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class BootstrapPhaseDetector {
+    
+    private final EpochParamStorage epochParamStorage;
+    private final EraGenesisProtocolParamsUtil eraGenesisProtocolParamsUtil;
+    private final StoreProperties storeProperties;
+
+    public boolean isInConwayBootstrapPhase(int epoch) {
+        boolean result = true;
+
+        if (isPublicNetwork()) {
+            Optional<EpochParam> epochParamOpt = epochParamStorage.getProtocolParams(epoch);
+
+            if (epochParamOpt.isPresent()) {
+                var protocolParams = epochParamOpt.get().getParams();
+                if (protocolParams.getProtocolMajorVer() >= 10) {
+                    result = false;
+                }
+            }
+        } else {
```

Comment:
@Sotatek-HuyLe3a Do you think we can enhance this logic so that it supports any public network or private network that started before the Conway era?

Can we try checking the era table through the EraService?